developerportal/templates/organisms/newsletter-signup.html

```diff
@@ -3,7 +3,13 @@
 <div class="mzp-l-content">
   <aside id="newsletter" class="newsletter-signup mzp-c-newsletter">
     <div class="mzp-c-newsletter-image">
-      <img src="{% static 'img/newsletter-promo.jpg' %}" alt="Newsletter signup promotional image">
+      <img
+        srcset="
+          {% static 'img/promo/newsletter@1x.jpg' %},
+          {% static 'img/promo/newsletter@2x.jpg' %} 2x"
+        src="{% static 'img/promo/newsletter@2x.jpg' %}"
```

Comment:
I am wondering, do we really want to fallback to the 2x asset for browsers that do not understand `srcset`? I believe generally browsers that do not support `srcset` will be older and likely be users with bandwidth-constrained connections.